src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -44,6 +44,34 @@ class databaseInfo extends resourceInfo {
         this.documentsCount(databaseInfo.DocumentsCount);
         this.indexesCount(databaseInfo.IndexesCount);
     }
+
+    static empty(resourceName: string): databaseInfo {
```

Comment:
What if I changed the database document (which will trigger a "Write")? This shouldn't "wake up" the database